### ISSUE TITLE
Empty path and root path should be handled the same

### ISSUE THREAD

**Issue by nexus-uw** [angular/router#171]
Currently, if I want the 'home page' to redirect to another view, I need to configure 2 routes, to achieve this:
```javascript
$router.config([     
   { path : '' , redirectTo : '/home' },   
   { path :  '/' , redirectTo: '/home' },   
   { path : '/home' , component : 'home' }
]);
```

I believe that $router should quietly treat path='' as path='/'.

**Comment by gdi2290** [comment]
it's probably due to [RouteRecognizer](https://github.com/tildeio/route-recognizer)

**Comment by nexus-uw** [comment]
thanks, I will look at what is going on over there.

**Comment by btford** [comment]
Still, the new router should be able to normalize before handing things off
to route recognizer.

I think it's worth keeping this issue open.

**Comment by gdi2290** [comment]
I think you can add support by updating [getCanonicalUrl router.ats#L185](https://github.com/angular/router/blob/v0.4.0/src/router.ats#L185) if I'm not mistaken

**Comment by nexus-uw** [comment]
I have submitted https://github.com/angular/router/issues/171 as a fix for this issue.

**Comment by btford** [comment]
This fix landed in 6d51f49f5d35554dcb4e51335f42d138a95498a3, and is available as part of v0.5.0.
